packages/openapi-fetch/test/index.test.ts

```diff
@@ -187,13 +190,14 @@ describe("client", () => {
         it("allows UTF-8 characters", async () => {
           const client = createClient<paths>();
           mockFetchOnce({ status: 200, body: "{}" });
-          const post_id = "post?id = 🥴";
           await client.GET("/blogposts/{post_id}", {
-            params: { path: { post_id } },
+            params: { path: { post_id: "post?id = 🥴" } },
           });
 
           // expect post_id to be encoded properly
-          expect(fetchMocker.mock.calls[0][0]).toBe(`/blogposts/${post_id}`);
+          expect(fetchMocker.mock.calls[0][0].url).toBe(
+            `/blogposts/post?id%20=%20🥴`,
```

Comment:
Side-effect of using `new Request()`—we’re not encoding anything manually. Seems OK? Still tests UTF-8 sticks